### ISSUE TITLE
After image capture or image selection blank black screen is appearing

### ISSUE THREAD

**Issue by rsgone** [Beelphegor/cordova-camscanner#2]
Hello Geeks,

After installing plugin I tried to execute android app, but its appearing blank black screen after selecting image from gallery or after capturing image by camera. It means it is doing nothing after passing "srcUri" to "cordovaCamscanner.scan" function.

If we jump into detail, there is a java file called "CamscannerActivty.java". it is executing all lines of onCreate() method, but doing nothing after that.(I did testing of each line execution by putting logs).

Looking for solution, Thank in advance.

Regards,
R.S.Gone

**Comment by Beelphegor** [comment]
Hi, do you have Camscanner Lite (free) installed?

**Comment by rsgone** [comment]
No, Not Installed.

**Comment by Beelphegor** [comment]
Having a version of Camscanner installed is required for the plugin to work, can you install it and see what happens? Maybe that validation is missing.

**Comment by rsgone** [comment]
Yes, I tried after installing your free version app too. No change, same result this time too

**Comment by saormart** [comment]
What kind of API key are you using ? If it's a trial, check if it's not expired!
I get a trial from here: [https://dev.camscanner.com/developers/status](https://dev.camscanner.com/developers/status)

**Comment by sagar2914** [comment]
Hi, I have face same problem plz help!!!!

**Comment by Xiralv** [comment]
Hi Im having the same problem too is this issue fix?

**Comment by veeran-dev** [comment]
Hi, any solutions found, i too face the same problem.
